src/main/java/bio/terra/testrunner/runner/VersionScriptResult.java

```diff
@@ -1,49 +1,129 @@
 package bio.terra.testrunner.runner;
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.List;
 
 @SuppressFBWarnings(
     value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD",
     justification = "This POJO class is used for easy serialization to JSON using Jackson.")
 public class VersionScriptResult {
+  public List<HelmVersion> helmVersions;
+  public List<GitVersion> gitVersions;
   // TODO: add more fields here for other version information we may care about
-  //  (e.g. git hashes, helm versions for other services)
-  public String wsmHelmAppVersion;
-  public String wsmHelmChartVersion;
 
-  // TODO QA-1643: Re-enable importComponentVersions API route pending DevOps readiness
-  // public Map<String, Map<String, String>> kubernetesComponentVersions;
+  // public VersionScriptResult() {}
 
   public VersionScriptResult(Builder builder) {
-    this.wsmHelmAppVersion = builder.wsmHelmAppVersion;
-    this.wsmHelmChartVersion = builder.wsmHelmChartVersion;
-    // this.kubernetesComponentVersions = builder.kubernetesComponentVersions;
+    this.helmVersions = builder.helmVersions;
+    this.gitVersions = builder.gitVersions;
   }
 
   public static class Builder {
-    private String wsmHelmAppVersion;
-    private String wsmHelmChartVersion;
-    // private Map<String, Map<String, String>> kubernetesComponentVersions;
+    private List<HelmVersion> helmVersions;
+    private List<GitVersion> gitVersions;
 
-    public Builder wsmHelmAppVersion(String wsmHelmAppVersion) {
-      this.wsmHelmAppVersion = wsmHelmAppVersion;
+    public VersionScriptResult.Builder helmVersions(List<HelmVersion> helmVersions) {
+      this.helmVersions = helmVersions;
       return this;
     }
 
-    public Builder wsmHelmChartVersion(String wsmHelmChartVersion) {
-      this.wsmHelmChartVersion = wsmHelmChartVersion;
+    public VersionScriptResult.Builder gitVersions(List<GitVersion> gitVersions) {
+      this.gitVersions = gitVersions;
       return this;
     }
 
-    // TODO QA-1643: Re-enable importComponentVersions API route pending DevOps readiness
-    // public Builder kubernetesComponentVersions(
-    //    Map<String, Map<String, String>> kubernetesComponentVersions) {
-    //  this.kubernetesComponentVersions = kubernetesComponentVersions;
-    //  return this;
-    // }
-
     public VersionScriptResult build() {
       return new VersionScriptResult(this);
     }
   }
+
+  // TODO: add more overloaded add methods for other version sources
+
+  // Builder for helm versions
+  public static class HelmVersion {
```

Comment:
nit: In keeping with your general cleanup here and separating POJO classes into their own files, could you also move `HelmVersion` and `GitVersion` into separate classes? Perhaps they can go in a new `runner.version` package. I think that will probably get refactored when we make the `VersionScriptResult` class overridable by different `VersionScript`s, but this is good for now.

Reply:
Refactored `HelmVersion, GitVersion` into separate classes.